### ISSUE TITLE
Make TextRow Title Multiline or in Two Lines

### ISSUE THREAD

**Issue by amsubhan** [xmartlabs/Eureka#2175]
I have a use case in which my title is for text row is large and it almost covers more than half of the screen. I want to ask that how can I make my title of TextRow in two lines?

Thanks!

**Comment by onuryurtturk** [comment]
Have you tried this? 'cell.textLabel.numberOfLines = 2'

**Comment by amsubhanTV** [comment]
> Have you tried this? 'cell.textLabel.numberOfLines = 2'

Yes I have tried this and set it 0 but it still shows in one line

**Comment by LeXuanKhanh** [comment]
try to set up an custom row, the default TextRow title doesn't support Multiline row

**Comment by iadwait** [comment]
@amsubhan @amsubhanTV 
To do this make sure you also **add \n** in your title string otherwise numberoflines parameter won't consider it.

Example Code:
<img width="465" alt="Screenshot 2024-09-26 at 3 49 04 PM" src="https://github.com/user-attachments/assets/82d5a011-6aed-4894-96c0-78e109c37cb1">

Output:
<img width="302" alt="Screenshot 2024-09-26 at 3 49 42 PM" src="https://github.com/user-attachments/assets/3fe1b869-6dd0-4372-be19-3d65202cadde">

Hope this fix your issue.
